### ISSUE TITLE
Difficulties when using a screen reader

### ISSUE THREAD

**Issue by DavidBitSource** [jitsi/jitsi-meet#6089]
When using Jitsi Meet with a screen reader, there are several unlabelled elements and it is not possible to determine the status of the microphone and camera or screen sharing. You can verify these issues e.g. using [NVDA screen reader for Windows](https://www.nvaccess.org/).

**Comment by Echolon** [comment]
Hi,
The team is aware of accessibility. Any reports are now grouped in the pinned issue #6090 and #3590.

I'm a community contributor and not directly affiliated with the Jitsi team.

**Comment by missmatsuko** [comment]
@l3d00m Just wanted to check if this issue should be reopened as well? Looks like the rest of the issues in the old accessibility meta issue have been tagged and reopened, while this one has only been tagged. Not sure if missed or intentional.
